android/CrashlyticsPlugin.java

```diff
@@ -15,15 +15,23 @@
 import com.tealeaf.logger;
 
 import com.crashlytics.android.Crashlytics;
-import com.crashlytics.android.core.CrashlyticsCore;
 import io.fabric.sdk.android.Fabric;
 
+
 public class CrashlyticsPlugin implements IPlugin {
 
   Activity _activity;
 
   public void onCreateApplication(Context applicationContext) {
+    logger.log("{crashlytics} registeriing crashlytics");
+      Fabric.Builder fb = new Fabric.Builder(applicationContext)
+            .kits(new Crashlytics(), new CrashlyticsNdk());
```

Comment:
Alex you had mentioned, CrashlyticsNdk will need 64 bit support.
So is it safe to go with this ?

Reply:
You are right, it is not safe. Pushed commit